### DISCUSSION TITLE
Catppuccinifier

### DISCUSSION THREAD

**Discussion by lighttigerXIV** [catppuccin/catppuccin#1960] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the suggested port be themed?

Is not a theme but it's a implementation of the LUTS to make wallpapers with the flavors filter. Since on wallpapers there's always a lot of people asking i decided to make this and try to make it a official way. I just need to make the readme more in line with others.

### Any additional comments?

I intend to do a Gui for it in rust as another option.

https://github.com/lighttigerXIV/catppuccinifier/tree/master

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by backwardspy** [comment]
hey! i know there's been some discussion on discord already but i want to leave my thoughts here in the interest of transparency.

as this is not a port per our usual definition, i feel this would be best maintained as a separate app that we could add to the showcase section of this repo.

we do also have a toolbox repo that this _could_ fit into, however those tools are generally more geared towards assisting with port development.

i'm interested in what other members of @catppuccin/staff think about this.

**Reply by lighttigerXIV** [reply]
I don´t mind having it in showcase. But maybe it does not belong anywhere 😂

**Comment by spookyintheam** [comment]
This seems like it would be a good fit in the toolbox repo. It would also make sense to rework/refactor this a bit better and see if it could be moved to pypi itself.

**Reply by lighttigerXIV** [reply]
Refactoring what? XD

**Reply by spookyintheam** [reply]
Refactoring in the sense to make it installable using `pipx` or `pip`. I'm not well-versed with how pypi packages work exactly so will need to look into that. Hope it makes sense.

**Comment by sgoudham** [comment]
Hi there @lighttigerXIV 👋, 

We had some more discussion about this (apologies, I think we'll need to aim for more transparent discussion on GitHub!). We came to the conclusion that in its current state, it wouldn't really be applicable in the showcase section or as a standalone port. 

From a quick look at the code, we can see that it's essentially wrapping a system call to `magick` for each flavour. We agreed that this python code adds more complexity than running the command itself. I actually think the concept of this repository would be better contained in the `catppuccin/wallpapers` repository itself as a list of steps in a CONTRIBUTING or README. 

If the repository was to be built out into a GUI app with more functionality for controlling the generation of the wallpapers, I think at that point it would be eligible for the showcase section (similar to the existing web app [faerber](https://farbenfroh.io/faerber) listed in the showcase already)

Hope the above makes sense! ❤️

**Reply by lighttigerXIV** [reply]
Chill. When i have time i will try to make the gui.

**Comment by sgoudham** [comment]
Closing as it's not a port request.
